.github/workflows/ci-docker.yaml

```diff
@@ -13,10 +13,13 @@ jobs:
     steps:
       - id: set-matrix
         run: |
-          echo "matrix={\"include\":$(echo "${{ vars.DOCKER_BUILDS }}" | yq -o=json -I 0 e '.branches.${{ github.ref_name }} // [{"name":"magfest/ubersystem:${{ github.ref_name }}", "plugins":[]}]' -)}" >> $GITHUB_OUTPUT
```

Comment:
Probably `magfest/ubersystem` should be the default upstream, otherwise we should probably figure out how to auto-lowercase the repo name and make it dynamic.